style.css

```diff
@@ -35,6 +35,10 @@ body{
     border-radius: 5%;
  
 }
+/* 
```

Comment:
I really like the look of the webpage especially the choice of color, and hover effect. 
Well done! the only I can suggest is, maybe you can consider deleting the commented out code in your css file (to make it cleaner), since you don't use it. I can understand if you keep it for learning purposes. Also, you can give more style to the paragraph :)

Reply:
Thankyou so much for you like it. Actually the reason ,why i didn't delete the comment that i want to show that i tried these ways too. and also for me to remember the thing what i read. 
now i used this extension.
i also done modification with colors that when its '+', color is green and '-' color is red and 0 will be always black.